c2corg_ui/static/partials/contexthelp/outing-avalanche_signs.html

```diff
@@ -0,0 +1,22 @@
+<p translate>Observations of danger signs, fresh avalanche trace, and natural avalanche or human-triggered avalanche the day of the outing.</p>
+<p translate>Give your observations concerning the snowpack stability:</p>
+<ul>
+<li><strong translate>Danger signs</strong> :</li>
+	<ul>	
```

Comment:
This HTML is not valid: you cannot open a new ``<ul>`` as a child of another ``<ul>``, it should probably be part of the ``<li>`` above. Same problem below.